charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 18.0.0
+version: 18.0.2
```

Comment:
adding new feature should raise chart minor version

Reply:
@monotek , thanks for the feedback. updated the minor version.